x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_columns.tsx

```diff
@@ -435,46 +494,17 @@ export const useColumns = (
       },
       ['Log rate change']: {
         'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnLogRateChange',
-        name: (
-          <>
-            <FormattedMessage
-              id="xpack.aiops.logRateAnalysis.resultsTable.logRateChangeLabel"
-              defaultMessage="Log rate change"
-            />
-            &nbsp;
-            <EuiIconTip
-              size="s"
-              position="top"
-              color="subdued"
-              type="questionInCircle"
-              className="eui-alignTop"
-              content={logRateChangeMessage}
-            />
-          </>
-        ),
+        name: LogRateColumnName,
+        sortable: isGroupsTable
+          ? undefined
+          : ({ doc_count: docCount, bg_count: bgCount }: SignificantItem) => {
+              if (logRateChangeNotAvailable) return NOT_AVAILABLE;
+              const logRateChange = getLogRateChangeValues(docCount, bgCount);
+              return logRateChange.sortableValue ?? logRateChange.factor;
```

Comment:
The sorting on 'log rate change' looks wrong here - it seems to be using `deviationBucketRate + baselineBucketRate` even when the baseline rate > 0.

<img width="798" alt="Screenshot 2024-09-23 at 16 55 29" src="https://github.com/user-attachments/assets/4ece66d4-f853-427d-b3eb-69f4a264097e">

Reply:
Fixed in c034dc521b8bd86f3d3fa65dbe8c2363ac7032a9